src/main.cpp

```diff
@@ -32,6 +32,31 @@ void hkOnTouchMove(void* _, SCallbackInfo& cbinfo, std::any e) {
     cbinfo.cancelled = g_pGestureManager->onTouchMove(ev);
 }
 
+void hkOnSwipeStart(void* _, SCallbackInfo& cbinfo, std::any e) {
+    HyprlandAPI::addNotification(PHANDLE, "[hyprgrass] swipe started", CColor{1.0, 0.2, 0.2, 1.0}, 500);
+    /*
+    auto gev = std::any_cast<DragGestureEvent&>(e);
```

Comment:
I'm trying to call `handleWorkspaceSwipe()` from the hook, but hyprland crashes with this cast.

Reply:
@horriblename do you have any idea why this is happening?

Reply:
Ahh woopsie